src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/managingofficer/ManagingOfficerChange.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.managingofficer;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.Change;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.managingofficer.officer.Officer;
+
+public abstract class ManagingOfficerChange<T extends Officer> extends Change {
+    private final String CHANGE_NAME = "officerAppointmentChange";
+    private final String APPOINTMENT_TYPE = "Managing Officer";
```

Comment:
This will now again be specific to sub-class. While same in CHIPS, needed as used to serialise the JSON/